plugin/integration_tests/test_plugin.py

```diff
@@ -37,7 +36,7 @@ def setUp(self):  # noqa
         # build blueprint path
         blueprint_path = os.path.join(os.path.dirname(__file__),
                                       'blueprint', 'blueprint.yaml')
-        with open('inputs.yaml') as f:
+        with open('inputs_plugin.yaml') as f:
```

Comment:
these values will be injected by the testing framework. for now this is ok, I'll email you how to implement the solution fitting the framework.